generateBuildrootSBOM.py

```diff
@@ -72,48 +75,54 @@ def create_buildroot_sbom(args, br_bom: cyclonedx.model.bom):
     return br_bom_local
 
 
-def main():
+def my_main():
     parser = argparse.ArgumentParser(description='CycloneDX BOM Generator')
     parser.add_argument('-i', action='store', dest='input_file', default='manifest.csv')
     parser.add_argument('-o', action='store', dest='output_file', default='buildroot_IOT_sbom')
     parser.add_argument('-n', action='store', dest='input_name', default='unknown')
     parser.add_argument('-v', action='store', dest='component_version', default='unknown')
 
     args = parser.parse_args()
-    print('Input file: ' + args.input_file)
-    print('Output BOM: ' + args.output_file)
+    print('Buildroot manifest input file: ' + args.input_file)
+    print('Output SBOM: ' + args.output_file)
     print('SBOM Component Name: ' + args.input_name)
     print('SBOM Component Version: ' + args.component_version)
 
-    # TODO provide a way to specify the meta data of this BOM using cyclonedx-python-lib v4.0.0 schema
-    # which should be managed by cyclonedx.model.bom.BomMetaData() but apparently not yet ready
+    # TODO provide a way to specify the metadata of this BOM using cyclonedx-python-lib v4.0.0 schema
+    # need support to provide the user with the ability to specify the following for this BOM
     # authors
     # component
     # manufacture
     # license
 
-    # TODO determine if we need both a br_bom and a new_bom
-
-    br_bom = cyclonedx.model.bom.Bom()
-    new_bom = create_buildroot_sbom(args, br_bom)
+    br_bom = Bom()
+    new_bom = create_buildroot_sbom(args.input_file, br_bom)
+    br_meta = BomMetaData(manufacture=OrganizationalEntity(name="Acme Inc"))
```

Comment:
I highly recommend removing the `manufacture` dummy data hare.
It might not appeal users ;-)

Reply:
GOOD CATCH! Yes will do.